Source/RomanNumeral.c

```diff
@@ -7,12 +7,6 @@
 #include "RomanNumeral.h"
 #include <string.h>
 
-
-static char * Append(char *output, const char *toAppend)
-{
-   return output + sprintf(output, "%s", toAppend);
-}
-
 void RomanNumeral_Convert(unsigned input, char *output)
 {
    char I[] = "I";
```

Comment:
strcat takes in two strings and I wasn't sure if it would be able to do the proper casting (and null terminating!) for a char.